pom.xml

```diff
@@ -18,14 +18,17 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <uprotocol-core-api.version>1.5.2</uprotocol-core-api.version>
```

Comment:
I just released 1.5.3 so please use that version. 1.5.3 includes the missing legal information for this open source project